Sources/Logging/Strings/StoreKitStrings.swift

```diff
@@ -225,6 +227,10 @@ extension StoreKitStrings: LogMessage {
 
         case let .skunknown_purchase_result(name):
             return "Unrecognized Product.PurchaseResult: \(name)"
+
+        case .sk2_sync_purchases_no_transaction_or_apptransaction_found:
+            return "syncPurchases was called, but we were unable to fetch any transactions or an AppTransaction " +
+            "from StoreKit 2."
```

Comment:
The same code path is used by `restorePurchases` so we could make this error a bit more generic:

```suggestion
        case .sk2_sync_purchases_no_transaction_or_apptransaction_found:
            return "Couldn't find previous transactions or an AppTransaction."
```